### PR TITLE
Creating Tecnologia container in Maquetado

### PR DESCRIPTION
**Pull request by SebasWolox**
## Summary
Adding Tech. container with its scss file.

## Trello Card
https://trello.com/c/rYt4f8vv/22-tecnolog%C3%ADa

### CODE REVIEW

File: Maquetado/index.html [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <img src="assets/triangle.svg" alt="triangle">
+        <h3>Front-End Tools</h3>
+      </header>
+      <div class="tech">
+        <div class="base">
```
Comment (meelrossi) [inline Maquetado/index.html]: Mmm i think one div Is enough. Try not ti wrap things in a div unless it's really necessary. Mayb hoy van merge all the Styles in one div

File: Maquetado/scss/tech.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  border-radius: 5px;
+}
+
+.tech {
+  height: 464px;
+  margin: auto;
+  padding: 16px;
+  width: 320px;
+
+  .base{
```
Comment (meelrossi) [inline Maquetado/scss/tech.scss]: Space☺️

File: Maquetado/scss/tech.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      padding: 16px;
+      width: 100%;
+
+      img {
+        height: 144px;
+        margin-bottom: 16px;
+      }
+
+      h4, span {
+        font-size: 20.8px;
```
Comment (meelrossi) [inline Maquetado/scss/tech.scss]: Mmm why decimals? Also try not to set styles to tags. Add them to a class

Comment (SebasWolox) [reply Maquetado/scss/tech.scss]: I did development in rems and when I did conversion, it returns that value. I am going to put 21px indeed 20.8px

File: Maquetado/index.html (hunk 1 above)
Comment (SebasWolox) [reply Maquetado/index.html]: screen shot 2018-11-26 at 9 11 46 am

I made this option using a container that contains card.

Comment (SebasWolox) [reply Maquetado/index.html]: screen shot 2018-11-26 at 9 35 20 am

I made this option using a container that contains card.

**Pull request merged**